### DISCUSSION TITLE
I logged into the server via SSH and then started a container using Podman. When I disconnected from SSH, the started container was terminated

### DISCUSSION THREAD

**Discussion by eonun** [containers/podman#23668] [Q&A]
Log in to the server via SSH, and then use Podman to start a container. When the SSH session is disconnected, the started container is terminated. How can I prevent the container from being terminated when the SSH session ends? Do I need to use podman generate systemd to create a service for keeping it alive, or is there a built-in service that can achieve this? Or is there a command that can accomplish this?

通过ssh登录到服务器，然后使用podman启动了容器，当断开ssh后，启动的容器就被结束了，怎么才能避免容器不会在ssh会话被终止时一同被终止，需要使用 `podman generate systemd` 创建一个服务来保活吗，还是有一个自带的服务可以实现？或者有什么命令可以实现这个？

**Comment by rhatdan** [comment]
You can use `loginctl enable-linger` to make user sessions last past user logout, but using a systemd service file like a quadlet is best if you want to run a container as a service.

**Comment by sbrivio-rh** [comment]
...or use `screen`, or `tmux`, or similar, so that your session remains alive and accessible after you disconnect (see https://hpc-wiki.info/hpc/Screen/tmux)?
